dockers/docker-platform-monitor/Dockerfile

```diff
@@ -15,4 +15,5 @@ ENTRYPOINT /usr/bin/config.sh       \
     && service lm-sensors start     \
     && service smartmontools start  \
     && service sensord start        \
+    && service fancontrol restart   \
```

Comment:
The below is fancontrol behavior:
1. Before starting, the fanctontrol will check whether it's pid is existed in '/var/run' or not. If pid is existed, the fancontrol won't start again. Otherwise, it will be started.

2. The '/var/run' of general linux is mounted as 'tmpfs'. Everything will be cleaned out (include pid) since system is rebooting. But in the docker, the '/var/run' is not mounted as 'tmpfs', the pid is still existed since system is rebooting.

So in the docker, we need to run service restart. When fancontrol is stopping, the pid will be removed and run successful in starting.